src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -238,6 +241,11 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 
 # VPN gateway
```

Comment:
tags is omitted for no change, empty string will clear the tags.  "Clearing the tags" however is too specific of language because we want no tags in the create but aren't clearing anything.  I'm open to any suggestions on wording.  Currently there is one tags text for all tags params.

---
In reply to: [74160956](https://github.com/Azure/azure-cli/pull/668#discussion_r74160956) [](ancestors = 74160956)